### ISSUE TITLE
[Bug] Screw the Review - Search Headers 

### ISSUE THREAD

**Issue by nys99** [mackenzieleach/screwthereview#100]
**Describe the Bug**
Please provide a clear and concise description of the bug you encountered.

Selecting $$$$ as a filter for search in Seattle, WA yields results with $$ instead of $$$$.

**Reproduce the Bug?**
What steps should be taken to reproduce the bug? ("Go to...", "Click on...", etc.)

Select Seattle, WA, and $$$$ and view the result. 

**Expected Behavior**
Please provide a clear and concise description of what behavior you expected.

The result should be a $$$$ restaurant, but it is a $$ restaurant. 

**Screenshots**
Please add any screenshots of the bug you encountered.

**Desktop:**

Chrome, Ubuntu 20.04

**Smartphone:**

**Additional Context**
Any other additional context for the bug?

**Comment by luuuk** [comment]
This appears to be caused by the front-end not setting the price header on requests to the server when the button is selected. @rachelye1 is probably the person to assign to this one?

**Comment by mackenzieleach** [comment]
This is the direct comment from peer review feedback and this issue should get addressed before we film final release tomorrow! I believe some of the confusion here lies in the buttons not staying highlighted despite the state being saved, and this is open in another issue. 

-"The layout of the website is easy to understand and navigate. There were a few issues that I 
encountered when using the site. The first was being unable to use a combination of filters for price and
distance. I also noticed that the search result did not always reflect the filters. For example, I put $$$$ 
but received a result of $$. Another thing I noticed is that when I selected the $$$$ filter, and thing 
selected a location, the $$$$ filter would unselect"
